src/response.rs

```diff
@@ -331,6 +335,26 @@ impl Response {
         self.version
     }
 
+    #[doc(hidden)]
+    pub fn set_peer_addr(&mut self, peer_addr: Option<impl std::string::ToString>) {
+        self.peer_addr = peer_addr.map(|addr| addr.to_string());
+    }
+
+    #[doc(hidden)]
+    pub fn set_local_addr(&mut self, local_addr: Option<impl std::string::ToString>) {
+        self.local_addr = local_addr.map(|addr| addr.to_string());
+    }
```

Comment:
This should be public and documented. Tide / Surf shouldn't expose this method, but `http-types` should.

```suggestion
    pub fn set_local_addr(&mut self, local_addr: Option<impl std::string::ToString>) {
        self.local_addr = local_addr.map(|addr| addr.to_string());
    }
```